lib/Catch2/.conan/build.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
```

Comment:
Could probably remove files from other build systems, bezel, Conan.  Also the other dot files and directories. not a big deal.